main/fopen_wrappers.c

```diff
@@ -81,60 +82,50 @@ PHPAPI ZEND_INI_MH(OnUpdateBaseDir)
 		return SUCCESS;
 	}
 
-	/* Otherwise we're in runtime */
-	if (!*p || !**p) {
-		/* open_basedir not set yet, go ahead and give it a value */
-		*p = ZSTR_VAL(new_value);
-		return SUCCESS;
-	}
-
 	/* Shortcut: When we have a open_basedir and someone tries to unset, we know it'll fail */
```

Comment:
I think it's more just a shortcut in the code as it is clear this will fail the check so we don't bother to do the actual check. Pretty much what it says so not sure what's confusing about this?